src/tools/rbd_ggate/Server.cc

```diff
@@ -103,10 +103,10 @@ void Server::wait_clean() {
 
   ceph_assert(!m_reader_thread.is_started());
 
-  std::lock_guard locker{m_lock};
+  std::unique_lock locker{m_lock};
 
   while (!m_io_pending.empty()) {
-    m_cond.Wait(m_lock);
+    m_cond.wait(locker);
```

Comment:
better off putting
```c++
m_cond.wait(locker, [this] {return m_io_pending.empty();});
```

Reply:
@tchaikov 
I was going for the quick fix, but fixing these makse sense as well.